BitwardenShared/Core/Platform/Services/TestHelpers/MockConfigService.swift

```diff
@@ -31,11 +34,13 @@ class MockConfigService: ConfigService {
     }
 
     func getFeatureFlag(_ flag: FeatureFlag, defaultValue: Bool, forceRefresh: Bool, isPreAuth: Bool) async -> Bool {
-        featureFlagsBool[flag] ?? defaultValue
+        let value = isPreAuth ? featureFlagsBoolPreAuth[flag] : featureFlagsBool[flag]
+        return value ?? defaultValue
     }
 
     func getFeatureFlag(_ flag: FeatureFlag, defaultValue: Int, forceRefresh: Bool, isPreAuth: Bool) async -> Int {
-        featureFlagsInt[flag] ?? defaultValue
+        let value = isPreAuth ? featureFlagsInt[flag] : featureFlagsInt[flag]
```

Comment:
⚠️ `featureFlagsIntPreAuth` should be used when is preAuth:
```suggestion
        let value = isPreAuth ? featureFlagsIntPreAuth[flag] : featureFlagsInt[flag]
```

Reply:
Thanks, good catch!